atomicapp/utils.py

```diff
@@ -377,6 +377,53 @@ def rm_dir(directory):
         distutils.dir_util.remove_tree(directory)
 
     @staticmethod
+    def getUid(user):
+        """
+        Get the UID of the specific user by grepping /etc/passwd unless
+        we are in a container.
+
+        Returns:
+            (int): User UID
+        """
+
+        # If we're in a container we should be looking in the /host/ directory
+        if Utils.inContainer():
+            passwd = os.path.join(HOST_DIR, "etc/passwd")
+        else:
+            passwd = "/etc/passwd"
```

Comment:
I think we aren't going to be able to use this. the user's mapping won't be in /etc/passwd if the user/grp information comes from ldap or some other identity management system. We'll have to use something like `getent passwd cdrage` to give us that information.